### DISCUSSION TITLE
Spotify in Deepin App Store

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3337] [Features Request & Ideas | 特性请求 & 头脑风暴]
Would be useful to be added Spotify in Deepin App Store. It’s one of the most popular Music app with millions of users.

**Comment by BLumia** [comment]
I guess it's currently not possible due to licensing issue.

Also: https://community.spotify.com/t5/Live-Ideas/Redistribute-Spotify-on-Linux-Distributions/idi-p/1695334/page/7
